modules/features/podcasts/src/main/java/au/com/shiftyjelly/pocketcasts/podcasts/viewmodel/PodcastEffectsViewModel.kt

```diff
@@ -51,6 +56,7 @@ class PodcastEffectsViewModel
         val podcast = this.podcast.value ?: return
         launch {
             podcastManager.updateTrimMode(podcast, trimMode)
+            trackPlaybackEffectsEvent(AnalyticsEvent.PLAYBACK_EFFECT_TRIM_SILENCE_AMOUNT_CHANGED, mapOf(PlaybackManager.AMOUNT_KEY to trimMode.analyticsVale))
```

Comment:
This event gets fired with an amount of "mild" when you toggle on trim silence from the podcast settings here, but if you turn on trim silence from the player there is no change event fired until you change from the default of mild. Can we make it behave consistently. 

I would probably lean toward not firing the event since it seems like it always defaults to mild, but I don't feel strongly at all.

Reply:
Makes sense to me 👍 
Updated in 7ee4fac